x-pack/plugins/observability/public/pages/slo_details/index.test.tsx

```diff
@@ -18,6 +18,10 @@ import { buildSlo } from '../../data/slo/slo';
 import type { ConfigSchema } from '../../plugin';
```

Comment:
More tests coming in a next PR?

Reply:
Yes, I'm even creating an issue to track that: https://github.com/elastic/kibana/issues/152568

Reply:
Awesome